### PR TITLE
Api/remove

### PR DESCRIPTION
**Pull request by ovekyc**
Fix conflict while rebase
same with PR before
Hmm i had to force push to remain comments.. Next time i will

### CODE REVIEW

File: src/test/node/items/controllers.js [hunk 1]
```diff
@@ -157,8 +157,9 @@ test('modify an item in database', t => {
   });
 });
 test('delete an item from database', t => {
+  const key = `item-${uuid()}`;
   const ops = [
-    { type: 'put', key: 'item1', value: itemRedSelo }
+    { type: 'put', key: key, value: itemRedSelo }
```
Comment (festinalente91) [inline src/test/node/items/controllers.js:162]: you can just type {key}
[https://github.com/airbnb/javascript#es6-object-concise](https://github.com/airbnb/javascript#es6-object-concise)

**Comment by festinalente91** [comment]
LG2M

**Comment by fguy** [comment]
LGTM

**Comment by festinalente91** [comment]
LGTM

**Pull request merged**